website/src/components/BlogsSlider/index.tsx

```diff
@@ -22,7 +23,11 @@ const BlogsSlider: React.FC<BlogsSliderProps> = ({ recommendedBlogs }) => {
   const { t } = useTranslation();
   const classes = useStyles();
   const mobileBreakpoint = VIEW_PORT.MOBILE_BREAKPOINT;
-
+  const history = useHistory();
+  const handleRedirectPath = (slug: any) => {
+    window.scrollTo(0, 0);
```

Comment:
@AVRahul  -- let's add the functionality of `auto-scrolling to the top` in `root of the application`.